libs/utils/WordPressUtils/src/main/java/org/wordpress/android/util/MediaUtils.java

```diff
@@ -508,6 +512,8 @@ public static String getDataColumn(Context context, Uri uri, String selection,
                     return cursor.getString(columnIndex);
                 }
             }
+        } catch (Exception errReadingContentResolver) {
```

Comment:
Do we really want to catch all exceptions here rather than just `SecurityException`?

Reply:
In theory yes, we should just catch `SecurityException`, and in practice too. Not sure why I put `Exception` there, I guess it was a mistake. Tested with `SecurityException` and it's catching the error on the test post.